python/python/glide/async_commands/sorted_set.py

```diff
@@ -160,6 +173,49 @@ def _create_zrange_args(
     return args
 
 
+def separate_keys(
+    keys: Union[List[str], List[Tuple[str, float]]]
+) -> Tuple[List[str], List[str]]:
+    """
+    Returns seperate lists of keys and weights in case of weighted keys.
+    """
+    if len(keys) == 0:
```

Comment:
```suggestion
    if not keys:
```

I am pretty sure this is enough